src/interfaces/resources/resourcesAddViewUI.py

```diff
@@ -0,0 +1,411 @@
+        self.cores.input_field.textChanged.connect(self.on_cores_edit)
+        self.ram.input_field.textChanged.connect(self.on_ram_edit)
+
+    def _init_verification_section(self):
+
+        section_layout = VerticalLayout(self.verification_section)
+
+        # title_frame
+        title_frame = SectionTitleFrame(self.verification_section, label_one_text="Resource Verification")
+
+        section_layout.addWidget(title_frame)
+        self.verification_hint = title_frame.get_label_two()
+
+        # content_frame
+        content_frame = Frame(self.verification_section, name="verification_content_frame")
+        section_layout.addWidget(content_frame)
+
+        content_layout = VerticalLayout(content_frame)
+
+        # ip_address
+        self.ip_address = TabsInputFrame(content_frame, title="IP Address:", title_width=66)
+        content_layout.addWidget(self.ip_address)
+
+    def _init_configuration_section(self):
+
+        section_layout = VerticalLayout(self.configuration_section)
+
+        content_frame = Frame(self.configuration_section, name="configuration_content_frame")
+        section_layout.addWidget(content_frame)
+
+        content_layout = VerticalLayout(content_frame)
+
+        # title frame
+        title_frame = SectionTitleFrame(content_frame, label_one_text="Machine Configuration")
+
+        content_layout.addWidget(title_frame)
+        self.configuration_hint = title_frame.get_label_two()
+
+        # frame: three config frame
+        frame = Frame(content_frame)
+        content_layout.addWidget(frame)
+        layout = HorizontalLayout(frame)
+
+        self.current_cpu_gpu = ConfigFrame(frame, label_one_text="Compute:", label_two_text="-")
+        layout.addWidget(self.current_cpu_gpu)
+
+        self.current_cores = ConfigFrame(frame, label_one_text="Cores:", label_two_text="-")
+        layout.addWidget(self.current_cores)
+
+        self.current_ram = ConfigFrame(frame, label_one_text="RAM:", label_two_text="-")
+        layout.addWidget(self.current_ram)
+
+    def _init_planning_section(self):
+
+        section_layout = VerticalLayout(self.planning_section)
+
+        # title frame
+        title_frame = SectionTitleFrame(self.planning_section, label_one_text="Resource Planning")
+
+        section_layout.addWidget(title_frame)
+        self.planning_hint = title_frame.get_label_two()
+
+        # frame: two line frame contains inputs
+        content_frame = Frame(self.planning_section, name="planning_content_frame")
+        section_layout.addWidget(content_frame)
+
+        content_layout = VerticalLayout(content_frame, space=18)
+
+        # line_frame: machine_name, cpu_gpu
+        line_frame = Frame(content_frame)
+        content_layout.addWidget(line_frame)
+        line_layout = HorizontalLayout(line_frame)
+
+        self.machine_name = TabsInputFrame(line_frame, title="Machine Name:", title_width=113, fix_width=True)
```

Comment:
is `113` used for any specific reason here? I'm hesitant to say we should be hardcoding width values due to various screen sizes.

Reply:
Well, I will say we can open another PR when we work on the screen resolution problem. This PR is just refactor the project structure of the old application. Almost everything would remain what we implemented previously. I know it is not perfect, but we should just go with it and fixed those problem later. Therefore, I will say, just leave the problem there, without resize the application, as soon as the application runs well, we should take it.